libraries/matrixui/src/main/kotlin/io/element/android/libraries/matrix/ui/messages/ToPlainText.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.matrix.ui.messages
+
+import io.element.android.libraries.matrix.api.timeline.item.event.FormattedBody
+import io.element.android.libraries.matrix.api.timeline.item.event.TextMessageType
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import org.jsoup.nodes.Node
+import org.jsoup.nodes.TextNode
+import org.jsoup.select.NodeVisitor
+
+fun TextMessageType.toPlainText() = formatted?.toPlainText() ?: body
+
+fun FormattedBody.toPlainText(prefix: String? = null): String? {
+    return this.toHtmlDocument(prefix)?.toPlainText()
```

Comment:
??? 🙄

Reply:
I guess I need to add some docs here 😅 . This is actually parsing the HTML of the formatted message and then getting the plain text value from it.

Reply:
Yeah, now In understand.
At first sight it seemed it converted text to html then back to text.

Perhaps `toPlainText` could be renamed to `parseHtml` ? That's what it does no?